src/Dofs/ConstraintHandler.jl

```diff
@@ -923,10 +925,22 @@ function _check_cellset_dirichlet(grid::AbstractGrid, cellset::Set{Int}, nodeset
     end
 end
 
+"""
+    create_symmetric_sparsity_pattern(dh::AbstractDofHandler, ch::ConstraintHandler, coupling)
```

Comment:
Off topic, but I think perhaps this functionality should be removed.

Reply:
👍 
I don't think I've ever used it

Reply:
Why?

Reply:
I wrongly thought that it was just a wrapper for `Symmetric`, but since it doesn't store all entries, it is faster. Maybe I should use it after all... 
On the other hand, just using `Symmetric(K)` gives a similar performance when solving
```julia
julia> dh = DofHandler(generate_grid(Quadrilateral, (100,100))); push!(dh, :u, 2); close!(dh);

julia> K = create_sparsity_pattern(dh);
julia> K.nzval .= rand(length(K.nzval));
julia> KS = create_symmetric_sparsity_pattern(dh);
julia> KS.data.nzval .= rand(length(KS.data.nzval));

julia> linsol(K,a) = K\a;
julia> f = rand(ndofs(dh));

julia> @btime linsol(Symmetric($K), $f);
  115.710 ms (90 allocations: 68.00 MiB)
julia> @btime linsol($KS, $f);
  117.943 ms (90 allocations: 54.95 MiB)
julia> @btime linsol($K, $f);
  148.528 ms (74 allocations: 65.69 MiB)

julia> @btime create_symmetric_sparsity_pattern($dh);
  5.400 ms (24 allocations: 17.90 MiB)
julia> @btime create_sparsity_pattern($dh);
  7.909 ms (24 allocations: 31.19 MiB)
  ```

Reply:
Fair point. Operations regarding the symmetry should not become bottlenecks during assembly, because we can symmetrize at the element level as in the incompressible linear elasticity example.

Reply:
I'll open a separate issue for discussing this.